LICENSE.txt

```diff
@@ -0,0 +1,2 @@
+Copyright (c) 2016 Gruntwork, Inc. All rights reserved.
+info@gruntwork.io
```

Comment:
Did you mean to add the same info as in the other repos?

Reply:
e.g. https://github.com/gruntwork-io/fetch-test-public/pull/2/files

Reply:
The other repos are using open source licenses. This repo is not meant to use an open source license.

Reply:
Aha makes total sense! Thank you :)